developerportal/templates/molecules/item-topic-link.html

```diff
@@ -16,4 +19,18 @@ <h4 class="no-underline">{{ topic.title }}</h4>
       {{ topic.description | richtext }}
     {% endif %}
   </div>
-</a>
+</a> {% endcomment %}
+
+<section class="mzp-c-card mzp-c-card-extra-small">
+  <a class="mzp-c-card-block-link" href="{{ topic.url }}">
+    <div class="mzp-c-card-content">
+      <img src="{% get_menu_item_icon topic %}" class="topic-item-icon" alt="" width="24" height="24">
+      <h2 class="mzp-c-card-title"><span>{{ topic.title }}</span></h2>
+      {% if topic.description %}
+        <div class="mzp-c-card-desc">
+        {{ topic.description | richtext }}
+        </div>
```

Comment:
Would it not be more semantic for this to be marked up as a `p` tag?

Reply:
Thanks for the suggestion.

The problem here is that the `richtext` filter outputs a `div` and/or `p` depending on what's in it, so we can't risk a wrapping `p`, alas